src/FITSIO.jl

```diff
@@ -147,6 +147,23 @@ include("header.jl")  # FITSHeader methods
 include("image.jl")  # ImageHDU methods
 include("table.jl")  # TableHDU & ASCIITableHDU methods
 
+"""
+    libcfitsio_version() -> VersionNumber
+
+### Purpose ###
+
+Return the version of the underlying CFITSIO library
+
+### Example ###
+
+```julia
+julia> using FITSIO
```

Comment:
I actually forgot to put `jldoctest` here. But yes, some [setup code](https://juliadocs.github.io/Documenter.jl/v0.8.3/man/doctests.html#Setup-Code-1) would be required for many of the doctests, like that of reading/writing functions etc to run